CONTRIBUTING.md

```diff
@@ -96,8 +96,9 @@ In this scenario, it’s possible that you found a bug or enhancement that we ha
 	2. Tests - All tests must pass before a merge of a pull request
 	3. Lint - All code must pass lint checks before a merge of a pull request
 13. Please never force push when a PR review has already started (because this messes with the PR review history)
-14. Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, and Android) to confirm the fix works as expected and there are no regressions.
-15. Please add a screenshot of the app running on each platform (Web, Desktop, iOS, Android, Mobile Web). If you are unable to build to iOS/OSX due to using a Windows machine, please let the reviewers know so they can double check that platform themselves.
+14. Please, pay attention to the pull request template, especially to how we require issues to be linked with the PR. We have processes in place to help us manage the open source issues and to pay contributors on time. If Github keywords such as `fixes` are used to link issue to PR, this will automatically close the related issue once the PR is merged and break our flow. Please, do not use these keywords and use format stated in the PR template.
```

Comment:
Suggesting shortening this up a bit, hope the point is still clear - let me know if you think I deleted any useful details!

```suggestion
14. Please pay attention to the pull request template, especially to how we link PRs with issues they fix. Please make sure you don't use Github keywords such as `fixes` in your PR description, as this can break our current automated steps for issue management. Please follow the PR template format carefully.
```